### ISSUE TITLE
Headings are not programmatically defined in the 'Retrospective' page

### ISSUE THREAD

**Issue by polatengin** [microsoft/vsts-extension-retrospectives#601]
Note: Same issue reproes throughout the application 

Impact on a Person with Disability (PWD):
Users who rely on screen readers who navigate the page using heading short cut key 'H' are facing difficulty in navigating as Headings are not programmatically defined in the 'Retrospective' page.

Test Environment:
OS: Windows 11 [Version 23H2 (OS Build 22631.2715)]
Browser: Edge Version (119.0.2151.97), Chrome Version (119.0.6045.200)
URL: [Work items - Boards (visualstudio.com)](https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/myactivity/)
Screen Readers: NVDA (2023.3), JAWS (2022.2110.6)

Repro Steps:
Step 1: Open MCAPS ADO in Edge/Chrome browser.
Step 2: Navigate to 'Retrospective' in left navigation and invoke it.
Step 3: Retrospective page gets displayed.
Step 4: Navigate using 'H' heading shortcut key.
Step 5: Verify whether Headings are defined in the 'Retrospective' page or not.

Actual Result: 
Headings are not programmatically defined in the 'Retrospective' page.
Current Observation: While navigating using heading shortcut key 'H', NVDA is narrating as 'No next heading' and upon pressing 'Shift + H', it is narrating as 'No previous heading'.

Expected Result:
Headings should programmatically define in the 'Retrospective' page.
Ex: Retrospective text should be defined as heading level h1.
What went well text should be defined as heading level h2.
What didn't go well text should be defined as heading level h2.

Ensure that:
1.All the text which is visually appearing as headings should be defined as a heading.
2.Main heading H1 should be defined in the page.
3.Multiple H1s should not be defined in the page.
4.Heading structure should be in sequential order like H1->H2->H3->H4->H5->H6.

Suggestive Fix:
1.Provide 'role="heading"' attribute and 'aria-level="1"' attribute for the element having class="text-2xl font-medium tracking-tight".
2.Provide 'role="heading" attribute and aria-level="2" attribute to the element having class="feedback-column-name".

MAS Reference:
[MAS 1.3.1 – Info and Relationships](https://aka.ms/MAS1.3.1)
 
WCAG Reference:
[https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fwww.w3.org%2fWAI%2fWCAG21%2fUnderstanding%2finfo-and-relationships.html&data=05%7c01%7cv-venkatab%40microsoft.com%7c152811a2bec143d2588808dad9b5b040%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638061671698366431%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=%2fEKPR2a5eIbXGSOloeJQVNFeOxwoYuSI2zFZJZfoRWs%3D&reserved=0)

AzDO Reference Link:
https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/edit/1713287
